discovery-provider/src/queries/health_check.py

```diff
@@ -50,3 +52,20 @@ def block_check():
         health_results,
         500 if error else 200
     )
+
+# Health check for latest play stored in the db
+@bp.route("/play_check", methods=["GET"])
+def play_check():
+    """
+       max_drift: maximum duration in seconds between `now` and the
+        latest recorded play record to be considered healthy
+    """
+    max_drift = request.args.get("max_drift", type=int)
```

Comment:
is there a default value? if we just hit play_check this would probably fail cause max_drift is None?